src/lib/grad.jl

```diff
@@ -16,23 +10,6 @@ macro nograd(ex)
   return blk
 end
 
-"""
-    nograd(x)
-
-Treats `x` as a constant. For example:
-
-```julia-repl
-julia> f(x) = sum(nograd(x) + x); gradient(f, randn(5))
-([1.0, 1.0, 1.0, 1.0, 1.0],)
-```
-
-In the example above, the gradient is an array of ones even though
-`f(x)` computes `sum(2x)`, because `nograd(x)` uses the value of `x`
-as if it were a constant.
-"""
-nograd(x) = x
-@nograd nograd
-
```

Comment:
why are these docstring gone?
Are these being deprecated ? If so why not be ising `@deprecate`/`@deprecate_binding` ?

Reply:
I don't think anyone was using them; `nograd` was only around temporarily and duplicated the existing `dropgrad`, which still exists.

Reply:
still we can add a deprecation